src/cdk/testing/protractor/protractor-element.ts

```diff
@@ -206,8 +206,8 @@ export class ProtractorElement implements TestElement {
   }
 
   /** Gets the value of a property of an element. */
-  async getProperty(name: string): Promise<any> {
-    return browser.executeScript(`return arguments[0][arguments[1]]`, this.element, name);
+  async getProperty<T = any>(name: string): Promise<T> {
+    return browser.executeScript<T>(`return arguments[0][arguments[1]]`, this.element, name);
```

Comment:
it seems that google's internal typings do not support a generic version of `executeScript` can you change this to `return browser.executeScript(...) as any`